### PR TITLE
History RFE abstract

### PR DESCRIPTION
**Pull request by jlarriba**

### CODE REVIEW

File: history.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Currently, every backup that is taken is called exactly the same. The outcome of this is that the newest backup always overwrites the previous one. ReaR, however, allows the usage of the configuration parameter ISO_PREFIX to change the name of the generated ISO.
```
Comment (jbadiapa) [inline history.md:1]: I understood the meaning of it, but I think this should be more high level description.
Something like:
The backup and restore feature currently does not set anything automatically to have several backups without overwritten them. (Have a historical list of the backups). The purpose of this is to configure ReaR to have them automatically.

Managing the historical backup list is not the purpose of this RFE.

File: history.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Currently, every backup that is taken is called exactly the same. The outcome of this is that the newest backup always overwrites the previous one. ReaR, however, allows the usage of the configuration parameter ISO_PREFIX to change the name of the generated ISO. 
+
+However, there are a set of other techniques that can be applied, jbadiapa is working on it, trying different capabilities.
```
Comment (jbadiapa) [inline history.md:3]: IMHO this last thing should be deleted.

Comment (jlarriba) [reply history.md:3]: That comment is there so you write a much more detailed spec with the solution you are applying to run the history. That's why this spec is almost empty: as there are different ways of configuring ReaR to achieve this, I did not want to propose something different that what you were implementing.

File: history.md (hunk 1 above)
Comment (jlarriba) [reply history.md:1]: Please, contribute to the spec with the ideas you gathered through the analysis you did while implementing this RFE.

**Pull request closed without merging**